package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "manifold.ml",
+  "name": "manifold",
```

Comment:
Did you also test whether publishing is working?
In addition, please make sure website publishing is working (run `yarn deploy`)

Reply:
if we publish, will it cause the npm the be updated ... how do we test it.